addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -356,9 +480,23 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * Updates `pendingGuestName` with the value of the input element
+         * referred by `guestNameInputRef`.
+         *
+         * @private
+         */
+        _updatePendingGuestNameWithInputValue() {
+            this.update({ pendingGuestName: this.guestNameInputRef.el.value });
+        }
```

Comment:
Do we really need a method for that? :thinking:

Reply:
I try to be as "prosaic" as possible in beginnings of flow, but you're right, this one is not necessary. I removed it :+1: